lib/wanda/messaging/mapper.ex

```diff
@@ -261,6 +301,11 @@ defmodule Wanda.Messaging.Mapper do
 
   defp map_value(value), do: %{kind: {:string_value, value}}
 
+  defp map_inner_value(value) when is_integer(value), do: {:int_value, value}
```

Comment:
comment: maybe I would put a comment on top of them explaining that these are not `Google.Protobuf.Value` values.
It took me a while to understand it.
Or rename to `map_check_custom_value`, so it is clear.
Yes, I would go with the renaming

Reply:
will rename to `map_check_custom_value`